### PR TITLE
Add localization messages for Turkish (tr) locale.

### PR DESCRIPTION
**Pull request by mertsayn**
I left default what I can't translate in English

### CODE REVIEW

**Comment by BingoRUS** [comment]
Hello @galetis,
Will it help you to translate the remaining strings if I provide descriptions (in some cases with screenshots) of each of them?

**Comment by mertsayn** [comment]
@BingoRUS I completed the remaining translates

File: js/localization/messages/tr.json [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        "dxColorView-ariaHex": "Renk kodu",
+        "dxTagBox-selected": "{0} seçili",
+        "dxTagBox-allSelected": "Tümü seçildi ({0})",
+        "dxTagBox-moreSelected": "{0} daha",
+        "vizExport-printingButtonText": "Yazdır",
+        "vizExport-titleMenuText": "Dışa Aktar/Yazdır",
+        "vizExport-exportButtonText": "{0} dosya",
+        "dxFilterBuilder-and": "Ve",
+        "dxFilterBuilder-or": "Veya",
+        "dxFilterBuilder-notAnd": "Değil",
```
Comment (BingoRUS) [inline js/localization/messages/tr.json]: Is it better to use this phrase here?
```suggestion
        "dxFilterBuilder-notAnd": "Değil Ve",
```

File: js/localization/messages/tr.json [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+        "dxScheduler-recurrenceRepeatCount": "olaylar",
+        "dxScheduler-allDay": "Tüm gün",
+        "dxScheduler-confirmRecurrenceEditMessage": "Yalnızca bu randevuyu veya tüm diziyi düzenlemek ister misiniz?",
+        "dxScheduler-confirmRecurrenceDeleteMessage": "Yalnızca bu randevuyu veya tüm diziyi silmek istiyor musunuz?",
+        "dxScheduler-confirmRecurrenceEditSeries": "Serileri düzenle",
+        "dxScheduler-confirmRecurrenceDeleteSeries": "Serileri sil",
+        "dxScheduler-confirmRecurrenceEditOccurrence": "Randevuyu düzenle",
+        "dxScheduler-confirmRecurrenceDeleteOccurrence": "Randevuyu sil",
+        "dxScheduler-noTimezoneTitle": "Saat dilimi yok",
+        "dxScheduler-moreAppointments": "{0} data",
```
Comment (BingoRUS) [inline js/localization/messages/tr.json]: I think there is a typo:
```suggestion
        "dxScheduler-moreAppointments": "{0} daha",
```

File: js/localization/messages/tr.json [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+        "dxPager-infoText": "Sayfa {0} / {1} ({2} veri)",
+        "dxPager-pagesCountText": "arasında",
+        "dxPivotGrid-grandTotal": "Genel Toplam",
+        "dxPivotGrid-total": "{0} Toplam",
+        "dxPivotGrid-fieldChooserTitle": "Alan Seçici",
+        "dxPivotGrid-showFieldChooser": "Alan Seçiciyi Göster",
+        "dxPivotGrid-expandAll": "Tümünü Genişlet",
+        "dxPivotGrid-collapseAll": "Tümünü Daralt",
+        "dxPivotGrid-sortColumnBySummary": "Sort \"{0}\" by This Column",
+        "dxPivotGrid-sortRowBySummary": "Sort \"{0}\" by This Row",
```
Comment (BingoRUS) [inline js/localization/messages/tr.json]: The `dxPivotGrid-sortColumnBySummary` and `dxPivotGrid-sortRowBySummary` messages look untranslated. Can you add right messages here please?

**Review by BingoRUS — CHANGES_REQUESTED** [review]

**Review by BingoRUS — APPROVED** [review]
@galetis Thanks for your contribution. Turkish localization will be available with the next DevExtreme updates: 19.2.3, 19.1.6 and 18.2.11.

**Pull request merged**